### PR TITLE
Service to send email

### PR DESCRIPTION
**Pull request by daisyzarzo**
relates to https://github.com/bespoken/source-api/issues/95

### CODE REVIEW

File: src/main/services/email/Notifier.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+export class Notifier {
+    private mailer: Mailer;
+
+    constructor() {
+        this.mailer = new Mailer();
+    }
+
+    public reportAlertToSales(userEmail: string, plan: string, allowedLogs: number, registeredLogs: number) {
+
+        const toAddress = (process.env.env === "prod") ? process.env.SALES_EMAIL : "daisy@bespoken.io";
```
Comment (ecruzado) [inline src/main/services/email/Notifier.ts]: Not sure if there is an email on dev for this case

Comment (jperata) [reply src/main/services/email/Notifier.ts]: use a different en variable or default to dev@bespoken.io

File: src/main/index.ts [hunk 2]
```diff
@@ -21,9 +22,18 @@ app.use(function (req, res, next) {
     next();
 });
 
-// Read environment variables from file
+// Read environment variables from
```
Comment (jperata) [inline src/main/index.ts]: revert change on this line

File: src/main/services/email/Notifier.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        const subject = `[Payment] ALERT Notification  - User exceeded the limits of the plan`;
+        const body = `<br>The  user with the email: <b>${userEmail}</b>  and  the subscribed plan: <b> ${plan}</b><br><br> ` +
+            `Has exceeded the limits of the subscribed plan:<br><br>` +
+            `   -   Allowed logs  for the plan : <b> ${allowedLogs}</b> <br>` +
```
Comment (jperata) [inline src/main/services/email/Notifier.ts]: remove extra space between logs and for

File: src/main/services/email/Notifier.ts [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+            `   -   Registered logs  for the user : <b> ${registeredLogs}</b> <br>`;
```
Comment (jperata) [inline src/main/services/email/Notifier.ts]: remove extra space between logs and for

**Pull request merged**